### PR TITLE
Fix adding textbox in the add_points method

### PR DESCRIPTION
**Pull request by yufeizhu600**
- [ ] Closes #xxxx 
 - [ ] Tests added 
 - [x] Tests passed 
 - [x] Passes ``git diff origin/master **/*py | flake8 --diff`` 
 - [x] Documented 

This PR fixed a bug that only apply the textbox parameters to the first point in the points_list in the add_points() function. Thanks to @peters77 for reporting the bug.
It also change the config keys 'list' and 'pt_size' in the config file for add_points() to make them consistence with the arguments of the function.
An example of using add_overlay_from_config() to apply overlays to a image is also added into the documents as suggested by @peters77.

### CODE REVIEW

File: pycoast/cw_base.py [hunk 1]
```diff
@@ -834,18 +834,18 @@ def add_overlay_from_dict(self, overlays, area_def, cache_epoch=None, background
...
             params = overlays['points'].copy()
 
-            points_list = list(params.pop('list'))
+            points_list = list(params.pop('points_list'))
             font_file = params.pop('font')
             font_size = int(params.pop('font_size', DEFAULT_FONTSIZE))
 
             symbol = params.pop('symbol', DEFAULT_SYMBOL)
-            pt_size = int(params.pop('pt_size', DEFAULT_PTSIZE))
+            ptsize = int(params.pop('ptsize', DEFAULT_PTSIZE))
```
Comment (djhoese) [inline pycoast/cw_base.py:842]: why this change? `point_size`?

Comment (yufeizhu600) [reply pycoast/cw_base.py:842]: It just make the keys from the config file consistence with the arguments in the function interface to avoid confusion. Similar as changing from 'list' to 'points_list' as we discussed.

Comment (djhoese) [reply pycoast/cw_base.py:842]: You changed all of them though. The kwarg here, the argument name in `add_points`, and the key in the config. With this change it isn't consistent with the cities.

Comment (yufeizhu600) [reply pycoast/cw_base.py:842]: I think the confusion is form that the keys ini config file for the add_cities() are not consistence with the add_cities() function. like 'list' vs 'citylist'; 'pt_size' vs 'ptsize'. i was trying to make they consistence in the add_points() case. If you think it is better to keep the 'pt_size' in the ini files, I can change them back.

**Comment by peters77** [comment]
Are there some news regarding this fix? Still waiting to use the new points feature with working text box backgound on all points. 👍

**Comment by djhoese** [comment]
Thanks @yufeizhu600!

**Comment by yufeizhu600** [comment]
Thanks for reviewing.@djhoese
